sklearn/linear_model/sag.py

```diff
@@ -131,7 +131,12 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1., beta=0.,
            *loss='multinomial'*
 
     alpha : float, optional
-        Constant that multiplies the regularization term. Defaults to 1.
+        Constant that multiplies the squared L2 regularization term.
+        Defaults to 1.
+
+    beta : float, optional
+        Constant that multiplies the l1 regularization term.  Only applied
```

Comment:
Can you replace the double spaces by a single space after each full stop.

Reply:
l1 -> L1 (to be consistent with your previous description)

Reply:
Done.

Reply:
but not pushed?